app/index.html

```diff
@@ -308,6 +303,15 @@ <h1 ng-bind="globalPage.title"></h1>
   </script>
   <!-- /build -->
 
-  <script type="text/javascript">_satellite.pageBottom();</script>
+  <!-- Adobe DTM -->
+  <script>
+    window.digitalData = {};
+  </script>
+  <!-- build:remove:dist -->
```

Comment:
I tried a few different plugins this afternoon.  What is the negative of just leaving both scripts in for development?  I don't think local devs will need DTM functioning.